README.md

```diff
@@ -58,14 +58,14 @@ Then you can install one of the following packages:
 
 #### On Windows
 
-Just drop the binary `rdiff-backup-VERSION-PLATFORM.exe`, possibly renamed to `rdiff-backup`,
-somewhere in your PATH and it should work, as it comes with all dependencies included.
+Just download and unpack the file `rdiff-backup-VERSION.winBITS.zip` from the 
```

Comment:
Dummy documentation. Could add reference to the "Asset" since it's collapsed.

Something like ... from the [Asset section in release page]